qt-models/diveplannermodel.cpp

```diff
@@ -894,8 +894,8 @@ void DivePlannerPointsModel::createPlan(bool replanCopy)
 				add_dive_to_trip(copy, current_dive->divetrip);
 			record_dive(copy);
 			QString oldnotes(current_dive->notes);
-			if (oldnotes.indexOf(QString(disclaimer)) >= 0)
-				oldnotes.truncate(oldnotes.indexOf(QString(disclaimer)));
+			if (oldnotes.indexOf(QString(disclaimer).left(40)) >= 0)
+				oldnotes.truncate(oldnotes.indexOf(QString(displayed_dive.notes).left(40)));
 			if (did_deco)
```

Comment:
I have some doubts if this check for "did_deco" is usefull here. We are at the moment doing it only in one out of three cases (save replanned dive as new). In the two other cases (plan new dive and save replanned dive) we always write the planner output to the notes, no mather if there was deco or not. Remove this "if" completely?